extensions/oidc-common/deployment/src/main/java/io/quarkus/oidc/common/deployment/devservices/keycloak/KeycloakDevServicesProcessor.java

```diff
@@ -83,13 +83,16 @@ public class KeycloakDevServicesProcessor {
     private static final String TENANT_ENABLED_CONFIG_KEY = CONFIG_PREFIX + "tenant-enabled";
     private static final String AUTH_SERVER_URL_CONFIG_KEY = CONFIG_PREFIX + "auth-server-url";
     private static final String PROVIDER_CONFIG_KEY = CONFIG_PREFIX + "provider";
-    // avoid the Quarkus prefix in order to prevent warnings when the application starts in container integration tests
-    private static final String CLIENT_AUTH_SERVER_URL_CONFIG_KEY = "client." + CONFIG_PREFIX + "auth-server-url";
```

Comment:
This one was introduced by @geoand to support the shared network cases, the actual integration test was removed but it would have to be restored, but users may be depending on it

Reply:
Yes, I checked that PR and there are no details I could make sense of. So I rewrote logic, I only set OIDC client properties when OIDC client is present and enabled; I only set OIDC properties when OIDC is present and enabled

I cannot imagine it could actually cause any problem and if so, let's have reproducer. I don't really believe that until I'll know more. Sorry.

Reply:
@geoand this is the PR in discussion https://github.com/quarkusio/quarkus/pull/21999

Reply:
My personal guess is that `CLIENT_AUTH_SERVER_URL_CONFIG_KEY` was about config validation messages and therefore my changes avoids that.

Reply:
I can't say I remember the details, sorry

Reply:
> I can't say I remember the details, sorry

I'd not expect you to. No worry, I think this comment was misread.

Reply:
I apologize for previous comments @sberyozkin . I invested time into that and misjudged that property. There is no need to do anything before this PR goes in as I'll restore previous behavior. I understand purpose of this property now.

Reply:
Hey @michalvavrik, there is absolutely no need to apologize :-), your help/time is super appreciated, we just discuss and challenge some comments as usual in a normal way, thanks